### PR TITLE
Site style changes & wording

### PR DESCRIPTION
**Pull request by natecode**

### CODE REVIEW

**Comment by natecode** [comment]
@jmgurney Give this site a try. I tried to simplify and remove duplicate fields, as well as vertical whitespace

File: content/index.html [hunk 1]
```diff
@@ -1,10 +1,13 @@
 ---
-title: BaySec
```
Comment (jmgurney) [inline content/index.html:2]: we should keep this, the main page now doesn't have a title

File: content/index.html [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
-description: SF Bay Area Security Professionals Meetup
```
Comment (jmgurney) [inline content/index.html:3]: We should probably keep this, but change to what you did below, this is useful for search engines.

File: gennext.sh [hunk 3]
```diff
@@ -31,7 +31,7 @@ esac
 
 cat <<EOF > "$dir/$(date -j -r $gensecs +"%Y-%m-%d")-$(date -j -r $esecs +"%B").html"
 ---
-title: $(date -j -r $esecs +"%B %Y") Baysec
+title: "Next BaySec: $(date -j -r $esecs +'%B %e'$ending', %Y')"
```
Comment (jmgurney) [inline gennext.sh:34]: Should we move this outside the link?

**Review by jmgurney — APPROVED** [review]
I added the dup fields back because they are kinda needed.. we maybe want to move that somewhere else, like a site definition, and use it from there.

**Pull request merged**